src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  users: User[];
+  onAdd: (todo: Omit<Todo, 'id'>) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ users, onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [titleHasError, setTitleHasError] = useState(false);
+
+  const [userSelect, setUserSelect] = useState(0);
+  const [userSelectHasError, setUserSelectHasError] = useState(false);
+
+  const isAllowedSymbols = (input: string): boolean => {
+    const pattern = /^[a-zA-Z А-ЩЬЮЯҐЄІЇа-щьюяґєії0-9]+$/;
+
+    if (!input) {
+      return true;
+    }
+
+    return !!input.match(pattern);
+  };
+
+  const clearTodoFields = () => {
+    setTitle('');
+    setUserSelect(0);
+  };
+
+  const handlerTitleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    // setTitle(event.target.value);
+    setTitle(currentTitle => {
```

Comment:
You have commented out a line of code. If this code is not needed, it is best to remove it to keep the code clean and easier to read.